author/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :first_name, :last_name, :id, :item
+
+  def initialize(first_name, last_name, id)
+    @first_name = first_name
+    @last_name = last_name
+    @id = id
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(item) unless @items.include?(item)
+    item.add_author(self)
```

Comment:
should add self as a property of the item object (by using the correct setter from the item object) this should be  like this:

` @item.author = self`